src/index.ts

```diff
@@ -61,17 +114,28 @@ export interface PushConfig {
   attibutes?: Map<string, string>;
 }
 
+export interface FlowControl {
```

Comment:
You should export this flow control and batching(bundling) from gax.

Reply:
tried finding but not able to, i checked with import google-gax module. need little help here.

Reply:
keep using this. as pubsub dont use gax flow control and batching. it has its own.